persistence-cassandra-3.11/pom.xml

```diff
@@ -10,13 +10,13 @@
   <artifactId>persistence-cassandra-3.11</artifactId>
   <properties>
     <!-- If you update this, make sure to keep `cassandra.bundled-driver.version` in sync -->
-    <!-- 3.11.13 depends on 3.0.1: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/3.11.13 -->
-    <cassandra.version>3.11.13</cassandra.version>
+    <!-- 3.11.14 depends on 3.11.3: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/3.11.14 -->
+    <cassandra.version>3.11.14</cassandra.version>
     <!--
       The driver used internally by cassandra-all for UDFs (must match the version declared in
       cassandra-all's POM).
     -->
-    <cassandra.bundled-driver.version>3.0.1</cassandra.bundled-driver.version>
+    <cassandra.bundled-driver.version>3.11.3</cassandra.bundled-driver.version>
```

Comment:
This is not a correct change:

![image](https://user-images.githubusercontent.com/10600041/211505828-cdae8dba-feb2-4435-a376-8399aa967e06.png)

Reply:
same for the 4.x btw, but I guess we want no 4.x related changes in this PR anyway

Reply:
I've fixed these versions based on the above.